test/test_einsum_autocast.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Dont see this being used anywhere?

Reply:
Sorry, artifact from local testing. Removed in next commit.